docs/nobrainer-guides/README.md

```diff
@@ -0,0 +1,31 @@
+# `nobrainer` Python API Guides
+
+This set of guides demonstrates basic usage of the `nobrainer` python API to train and use neural network models for neuroimaging. The `nobrainer` project was developed under the support of NIH RF1MH121885 and R01EB020470. It is distributed under the Apache 2.0 license.
+
+## Accessing the guides
+
+These guides are available in both jupyter notebook versions and as standalone python scripts.
+
+### Jupyter notebooks
+
+See the [notebooks](notebooks) directory for the guides in jupyter notebook format. These notebooks can be run for free on Google Colab, accessed [here](https://colab.research.google.com/github/neuronets/nobrainer-book/blob/ohinds-guide-api).
```

Comment:
this should be updated before this is merged.

Reply:
:+1: